<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HotWindowClass</class>
 <widget class="QWidget" name="HotWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>1839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HotWindowClass</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="netMusicHotLayout" rowminimumheight="0" columnminimumwidth="0">
    <property name="horizontalSpacing">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/netMusicHot.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/hotTitle.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/netMusicNew.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="netMusicNewLayout"/>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>990</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>82</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/baidu.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>1070</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="baiduLayout"/>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>550</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/tieba.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>630</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="tiebaLayout"/>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>550</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/weibo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>630</y>
     <width>261</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="weiboLayout"/>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/netMusicUp.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="netMusicUpLayout"/>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>1420</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/baidu_arts.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>1510</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="baiduArtsLayout"/>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>980</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/baidu_dongman.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>1070</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="baiduDongmanLayout"/>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>980</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/baidu_fiction.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>1070</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="baiduFictionLayout"/>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>1420</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/baidu_movie.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>1510</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="baiduMovieLayout"/>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>1420</y>
     <width>250</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/baidu_opera.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>1510</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="baiduOperaLayout"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HotWindowClass.qrc"/>
 </resources>
 <connections/>
</ui>
